src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -125,9 +130,16 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
           ElasticsearchSinkConnectorConfig.CONNECTION_USERNAME_CONFIG);
       final Password password = config.getPassword(
           ElasticsearchSinkConnectorConfig.CONNECTION_PASSWORD_CONFIG);
+      Boolean alwaysSecured = config.getBoolean(
+          ElasticsearchSinkConnectorConfig.CONNECTION_SSL_CONFIG);
+      alwaysSecured = alwaysSecured == null ? false : alwaysSecured;
```

Comment:
it can't be `null` (you may confirm with the code in `AbstractConfig` and `ConfigDef`). There's no value you can give to your properties that will result in a `null` value. I think the only way to have `null` here, would be to set `null` as your default (again, I'd have to remember if that's valid), which however you don't do above.

Reply:
I reread the code and I agree!